packages/replay-next/components/inspector/values/FunctionRenderer.tsx

```diff
@@ -73,3 +79,50 @@ export function functionProtocolObjectToString(protocolObject: ProtocolObject) {
 
   return `${functionName}(${functionParameterNames.join(", ")}) {}`;
 }
+
+type FunctionLocationRendererProps = {
+  object: ProtocolObject;
+};
+
+export function FunctionLocationRenderer({ object }: FunctionLocationRendererProps) {
```

Comment:
I think this component should be in its own file (in `replay-next/components/inspector/properties`)